### PR TITLE
CORE-1143: added the is_public flags to the app listings

### PR DESCRIPTION
**Pull request by slr71**

### CODE REVIEW

File: src/clients/permissions.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+export const getPublicAppIDs = async () => {
+    const reqURL = new URL(config.permissionsURL);
+    reqURL.pathname = `/permissions/subjects/group/${config.publicGroup}/app`;
+    const resp = await fetch(reqURL);
+    if (!resp.ok) {
+        const msg = await resp.text();
+        throw new Error(msg);
+    }
+    const resp_1 = resp;
```
Comment (ash3rz) [inline src/clients/permissions.js]: Is there a reason to create `resp_1` and not use `resp` directly?

**Review by ash3rz — APPROVED** [review]
I didn't catch anything 👍

File: src/clients/permissions.js (hunk 1 above)
Comment (slr71) [reply src/clients/permissions.js]: That's what I get for not reading an automated refactor before committing the change. 😊

**Comment by slr71** [comment]
Thanks for the review!

**Pull request merged**